{
  "modified": "2024-06-25T13:37:07Z",
  "published": "2024-06-25T13:37:07Z",
  "schema_version": "1.5.0",
  "id": "MAL-2024-5349",
  "summary": "Malicious code in manyhttp (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "manyhttp",
        "purl": "pkg:pypi/manyhttp"
      },
      "versions": [
        "2.33.15",
        "2.33.16",
        "2.33.14",
        "2.33.17",
        "2.33.18"
      ]
    }
  ],
  "credits": [
    {
      "name": "ReversingLabs",
      "type": "FINDER",
      "contact": [
        "https://www.reversinglabs.com"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "id": "RLMA-2024-04131",
        "import_time": "2024-06-28T02:49:24.371034286Z",
        "modified_time": "2024-06-25T13:37:07Z",
        "sha256": "3596d2e789daa6d136346943c1e823375749899e8cd617b3d972f683f21784e0",
        "source": "reversing-labs",
        "versions": [
          "2.33.15",
          "2.33.16",
          "2.33.14",
          "2.33.17",
          "2.33.18"
        ]
      }
    ]
  }
}
